authors/settings.py

```diff
@@ -137,7 +136,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
Hi Patrick. Well done so far. Why do you have the "DEFAULT_AUTHENTICATION_CLASSES" commented out?
If there is a valid reason for doing so. Its not proper to push commented out code to the remote.

Reply:
Thanks @e-ian, I've removed the unnecessary code as advised